docs/src/pages/tutorial/run-in-raspberry-pi.md

```diff
@@ -44,20 +45,42 @@ npm -v
 
 ![nodejs & npm version](/tutorials/raspberry-pi/node-version-and-npm-version.png)
 
-## Run Monika in Raspbian
+## Install and Run Monika in Raspbian
 
-You can install Monika by source using git clone and npm install:
+For now, the way to install Monika in Raspbian is to install from source code.
+
+There are two ways to get Monika in Raspbian, from archive file and via Git.
+
+- Download Monika archived installer
+
+```bash
+cd ~
+wget https://github.com/hyperjumptech/monika/archive/refs/tags/v1.5.1.tar.gz
+tar xf v1.5.1.tar.gz
+cd monika-v1.5.1
```

Comment:
To avoid updating this doc over and over again when we release a new version, get the latest tag first before downloading. Check my comment below how to.